@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -85,6 +85,7 @@ export default {
     i19reopenOrder: () => 'Reabrir pedido',
     i19transactionCode: () => i18n(i19transactionCode),
     i19ticketCode: () => i18n(i19ticketCode),
+    i19observationOrder: () => 'Observações do pedido',
```

Comment:
_observation order_ é algo como _ordem de observação_, então ainda não é coerente com a tradução...

```suggestion
    i19orderObservations: () => 'Observações do pedido',
```